REST/src/main/resources/static/js/board.js

```diff
@@ -16,23 +16,62 @@ function setDivsForBoard(board, clickable) {
         for (var j = 0; j < 10; j++) {
             var cell = document.createElement('DIV');
             cell.setAttribute("class", "cell col-xs-1 col-xs-push-1");
-            cell.setAttribute("id", "cell" + i + j);
-            cell.setAttribute("value", i + "," + j);
+
+            var attribute = (j.toString()).concat(i.toString());
+
+            setAttribute(cell, attribute, myBoard);
```

Comment:
maybe different name to recognize which method is used because setAttribute is node method

Reply:
done